From 1411607f4fc7e4135eef410207dc8af33d3b3191 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Fri, 29 Jun 2018 14:16:39 +0800
Subject: [PATCH 1325/1345] arm64: dts: marvell: remove redundant pcie dts
 nodes

Because there has been pcie node in mainline upstream, so delete
unnecessary ones from sdk patch.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dts |    4 ----
 arch/arm64/boot/dts/marvell/armada-8040-db.dts |   13 ++++---------
 arch/arm64/boot/dts/marvell/armada-8040.dtsi   |    9 ---------
 3 files changed, 4 insertions(+), 22 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index ca3dd55..7d9998a 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -173,10 +173,6 @@
 				};
 			};
 		};
-
-		pcie@0x640000 {
-			status = "okay";
-		};
 	};
 
 	usb3h0_phy: usb3-h0-phy {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index 1920819..959486a 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -120,12 +120,6 @@
 				 */
 			};
 		};
-		pcie@0x600000 {
-			status = "okay";
-		};
-		pcie@0x640000 {
-			status = "okay";
-		};
 
 		cpm_usb3h0_phy: cpm_usb3_phy0 {
 			compatible = "usb-nop-xceiv";
@@ -274,6 +268,10 @@
 	status = "okay";
 };
 
+&cpm_pcie0 {
+	status = "okay";
+};
+
 /* CON5 on CP0 expansion */
 &cpm_pcie2 {
 	status = "okay";
@@ -488,6 +486,3 @@
 &cps_pcie1 {
 	status = "okay";
 };
-&cps_pcie2 {
-	status = "okay";
-};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040.dtsi b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
index 2b01a58..7a10963 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
@@ -220,15 +220,6 @@
 		axim-hb1-wr@3c9000 {
 			status = "disabled";
 		};
-		pcie@0x600000 {
-			msi-parent = <&gic_v2m0>;
-		};
-		pcie@0x620000 {
-			msi-parent = <&gic_v2m0>;
-		};
-		pcie@0x640000 {
-			msi-parent = <&gic_v2m0>;
-		};
 	};
 };
 
-- 
1.7.9.5

